dashboard.csss-ci.com/server/api/models/Households_model.php

```diff
@@ -71,6 +123,12 @@ public function women() {
                         ->result();
     }
 
+    /**
+     * Returns all records of children with desired association with additonal filter
+     *
+     * @param Array or String $where - Associative array or custom string
+     * @return Array containing rows with desired association
+     */
     public function child($where) {
         return $this->db->select_sum('child')
```

Comment:
you get the gist now:
- descr should mention sum and be more specific
- where should be more specific
- make sure you are using desired association correctly